README.md

```diff
@@ -1,3 +1,8 @@
 # git1_CDV
 
 ### About project in CDV for GIT TRAINING
+
+## I Like Cookies
+
+##GIT tutorial [link](https://www.udemy.com/course/kurs-git-i-github-od-podstaw/)
```

Comment:
## [ Link to GIT tutorial](https://www.udemy.com/course/kurs-git-i-github-od-podstaw/)